dockeradapter/dockeradapter.go

```diff
@@ -123,10 +124,23 @@ func ImageExists(ctx context.Context, client ImageInspector, imageName string) b
 func New() (Client, error) {
 	concreteClient := new(concreteDockerClient)
 	logging.Stderr("[Docker Adapter] Creating container runtime client: Docker")
+	ctx := context.Background()
 	dockerAPIClient, err := client.NewEnvClient()
 	if err != nil {
 		return nil, fmt.Errorf("[Docker Adapter] Unable to initialise container runtime type: Docker, error: %s", err)
 	}
+
+	serverVersion, _ := dockerAPIClient.ServerVersion(ctx)
+	clientVersion := dockerAPIClient.ClientVersion()
+	clientAPIVersion, _ := version.NewVersion(clientVersion)
+	serverAPIVersion, _ := version.NewVersion(serverVersion.APIVersion)
+
+	if clientAPIVersion.Compare(serverAPIVersion) == 1 {
+		return nil, fmt.Errorf(
+			"[Docker Adapter] Unable to initialize container runtime type: Docker, error: client is newer than server (client API version: %s, server API version: %s)",
```

Comment:
If `z` is the Puppet house style (sorta makes sense, :us: company), then I say we do that until we go full i18n. Happy to sumbit another commit to change `s` -> `z`